lib/src/main/kotlin/com/swmansion/starknet/data/NetUrls.kt

```diff
@@ -1,5 +1,3 @@
-@file:JvmName("Nets")
```

Comment:
Same here

Reply:
Actually it works normally, it's better to remove this.